.github/CODEOWNERS

```
@@ -556,3 +556,7 @@
 /adaptive_shield/README.md             @adaptiveshield @DataDog/documentation @DataDog/marketplace-review
 /typingdna_activelock/manifest.json    @TypingDNA/logintegrations @raulpopa @endresstefan @DataDog/documentation @DataDog/marketplace-review
 /typingdna_activelock/README.md        @TypingDNA/logintegrations @raulpopa @endresstefan @DataDog/documentation @DataDog/marketplace-review
+# Sofy
+/sofy_sofy/                                     @Sofy @DataDog/marketplace-review
+/sofy_sofy/*.md                                 @Sofy @DataDog/marketplace-review
+/sofy_sofy/manifest.json                        @Sofy @DataDog/marketplace-review
```

Could you please follow the same format as the other integrations for consistency?

1. Move `/sofy_sofy/ @Sofy @DataDog/marketplace-review` up under `# Community partners`
2. Make the following changes:
```suggestion
/sofy_sofy/*metadata.csv                        @Sofy @DataDog/documentation @DataDog/marketplace-review
/sofy_sofy/manifest.json                        @Sofy @DataDog/documentation @DataDog/marketplace-review
/sofy_sofy/README.md                            @Sofy @DataDog/documentation @DataDog/marketplace-review
```